variant-errors.go

```diff
@@ -17,3 +17,15 @@ func NewErrVariantNotFound(id int64) ErrVariantNotFound {
 		id,
```

Comment:
File name also needs to be `variant_errors.go`.

Reply:
All the files have dashes in them and if I change it to this then it would be inconsistent. Must have been in our in-between phase of doing it this way.

What do you think? accept the inconsistency or change them all.

Reply:
I think we could change it but it would be its own thing. Also there are tests going on which makes me worry.

Reply:
Better to follow what is already being done in repo I think, so as you have it is fine.

Reply:
Each repo is its own 💎